Makefile

```diff
@@ -215,6 +215,19 @@ _run_setup:
 		exit 1; \
 	fi
 	@mkdir -p logs
+	@poetry run uvicorn opendevin.server.listen:app --port $(BACKEND_PORT) &
+	@echo "$(YELLOW)Waiting for the app to start...$(RESET)"
+	@until nc -z localhost $(BACKEND_PORT); do sleep 0.1; done
+	@echo "$(GREEN)Application started successfully.$(RESET)"
+
+# Run the app
```

Comment:
The difference between "Run the app" and "Run the app (standard mode)" is not very clear, could you please make the comment more descriptive?